### DISCUSSION TITLE
Buat user baru untuk akses buku kas khusus

### DISCUSSION THREAD

**Discussion by utay-cugung** [buku-masjid/buku-masjid#67] [Ideas]
Pada kepengurusan masjid umumnya terdapat kepengurusan majlis yang di kelola oleh jamaah muslimah/ibu2 pengajian.
Sehingga perlu hak akses buku kas khusus untuk pengurus pengajian.

Akses khusus sebagai bendahara pengurus pengajian mungkin cukup. Atau disamakan dengan  pembagian akses seperti default juga lebih baik.

![Screenshot_20240408-162530_Edge](https://github.com/buku-masjid/buku-masjid/assets/155012206/66b5cffc-9875-4d0f-ad9e-7db1602d22b5)
![Screenshot_20240408-162337_Edge](https://github.com/buku-masjid/buku-masjid/assets/155012206/c21a4942-3bae-424e-9e1b-4bddac468590)

**Comment by nafiesl** [comment]
Halo mas @utay-cugung ok sepertinya bisa mas. Jadi kita akan coba terapkan seperti ini ya:
1. Tiap buku kas bisa dipilih user bendahara mana yang akan mengelola pembukuannya (input/edit transaksi dan kategori)
2. Bendahara tetap bisa melihat isi laporan semua buku kas.
3. Bendahara hanya bisa meng-edit/input transaksi pada buku kas yang ditugaskan kepadanya
4. User admin tetap bisa input data di semua buku kas.

**Reply by utay-cugung** [reply]
Pas banget rincian detailnya mas

**Reply by nafiesl** [reply]
Task untuk pengerjaan usulan ini #69

**Comment by nafiesl** [comment]
mas @utay-cugung untuk usulan ini kita sudah implementasikan melalui PR #72 (sudah merge). Silakan coba dilalukan git pull dan jalankan /usr/bin/php8.1 artisan migrate.
